PowerFGT/Public/cmdb/firewall/address.ps1

```diff
@@ -36,9 +36,15 @@ function Add-FGTFirewallAddress {
     #>
 
     Param(
-        [Parameter (Mandatory = $true)]
-        [ValidateSet("ipmask","fqdn")]
+        [Parameter (Mandatory = $false, DontShow)]
         [string]$type,
+        [Parameter (Mandatory = $false, ParameterSetName = "fqdn")]
+        [string]$fqdn,
+        [Parameter (Mandatory = $false, ParameterSetName = "ipmask")]
+        [ipaddress]$ip,
+        [Parameter (Mandatory = $false, ParameterSetName = "ipmask")]
+        [ValidateScript({($_ -match [ipaddress]$_) -or ($_ -in 8..32)})] #matches ipaddress and int
```

Comment:
0..32 ?

Reply:
Remove this part. This means the number must be betwee 8 and 32 (8..32).

Reply:
Ok, but it is a good idea (may be on seperate commit) with some test :)